tests/integration/component.go

```diff
@@ -816,18 +816,6 @@ func componentTests(args ...string) {
 	Context("odo component delete should clean owned resources", func() {
 		appName := "app"
```

Comment:
Can you please add a dynamic string instead of hardcode to avoid name collision in future. Something like 
```
appName = helper.RandString(5)
```